client/electron/src/setup.ts

```diff
@@ -213,6 +214,7 @@ export class ElectronCapacitorApp {
   }
 
   updateMountpoint(path: string): void {
```

Comment:
```suggestion
  createAndRegisterBaseMountpointFolder(path: string): void {
```

I suggest to rename the method to something that reflect what the method do and give more hint about the path value